app/routes/how-can-i-help.career.tsx

```diff
@@ -584,34 +584,43 @@ const FieldBuildingPath = () => (
   </>
 )
 
-const CareerPaths = () => (
-  <>
-    <p className="grey default padding-bottom-32">There are 3 major career paths in AI safety:</p>
-    <div className="flexbox padding-bottom-80">
-      <CardSmall
-        action="#research"
-        iconColor="var(--colors-teal-700)"
-        title="AI alignment research"
-        description="Research roles dedicated to solving the technical challenge of AI alignment, and non—technical supporting roles"
-        icon={Microscope}
-      />
-      <CardSmall
-        action="#governance"
-        iconColor="var(--colors-teal-700)"
-        title="AI governance & policy"
-        description="Roles focused on developing and implementing policies that guide AI development and usage"
-        icon={GovermentBuilding}
-      />
-      <CardSmall
-        action="#field-building"
-        iconColor="var(--colors-teal-700)"
-        title="AI safety field-building"
-        description="Roles that direct people or funding towards AI risk, rather than working on the problem directly"
-        icon={PuzzlePieces}
-      />
-    </div>
-  </>
-)
+const CareerPaths = () => {
+  const smoothScroll = (id: string) => {
+    const element = document.getElementById(id)
+    if (element) {
+      element.scrollIntoView({behavior: 'smooth'})
+      window.history.pushState(null, '', `#${id}`)
```

Comment:
@mruwnik I tried with remix's `navigate`, but it just jumped to the section. is `history.pushState` alright?

Reply:
how about just applying `scroll-behavior: smooth;` to the page?

Reply:
Cus I'm a silly guy, and claude didn't suggest it for some reason

Reply:
took me years to get to the stage where I always triple check for a css solution :D